src/TPS/CharacterController.js

```diff
@@ -6,7 +6,7 @@
 
   ns.CharacterController = function ( object3d, radius ) {
 
-    THREE.EventDispatcher.prototype.apply( this );
+    Object.assign( ns.CharacterController.prototype, EventDispatcher.prototype );

```

Comment:
Can we just assign `THREE. EventDispatcher .Prototype`?

Reply:
THREE.EventDispatcher.prototype was completely removed, in its place THREE.EventDispatcher.apply works in previous versions, but as of now, it states: EventDispatcher.apply has been removed. Inherit or Object.assign the prototype to mix-in instead.